src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpoint.java

```diff
@@ -461,6 +472,12 @@ private void validateMetadataSearchParameters(Map<String, String> queryParameter
                 throw new BadRequestException(String.format("unknown security classification '%s'", sc));
             }
         });
+
+        param(queryParameters, MetaData.SORT_DIRECTION).ifPresent(sd -> {
+            if (Stream.of("ASC", "DESC").noneMatch(direction -> direction.equalsIgnoreCase(sd))) {
```

Comment:
perhaps we could introduce a SortDirection enum as we have one for the SecurityClassification param?

Reply:
I looked at it but it didn't seem to add much value because of the limited places it would be used and because Metadata is passing strings around so you would have to convert to it again.

SecurityClassification is used in lots more places than this would be